### DISCUSSION TITLE
Better Themeing

### DISCUSSION THREAD

**Discussion by maxthorson** [santangeloChristian/Project-Hub#27] [Ideas]
I was wondering if you would consider redoing the theme, by things like default dark mode, pill shaped buttons, and more hover effects. I think that these would make the site seem a bit more modern. I think that rearranging some of the contents, or possibly adding an overflow menu after the home page. Basically, a ui redesign. I personally think somethign like google's  material design (Material Design and Material You), as well as some color theming across the site. Idk just and idea tho
![image](https://github.com/unbl0ck/unbl0ck.github.io/assets/116239673/f1db53ef-9f14-4b12-8a9b-541bc50a6e31)
![image](https://github.com/unbl0ck/unbl0ck.github.io/assets/116239673/e1605df6-f53f-463e-881d-9943fa4231af)

**Comment by IamChristianS** [comment]
Sorry for only talking about this now, but please, in the future, use one of the official methods of contact (i.e. Email, Google Forms, GitHub Issues). As for the UI Redesign, not much will be done about it. I understand the importance of having a nice-looking site, but I'm still learning UI and it's not my main priority. However, I am pleased to say that there is a site redesign planned. ;)
